kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -376,7 +377,9 @@ def test_port(self):
         assert self.expected_pod == actual_pod
 
     def test_volume_mount(self):
-        with mock.patch.object(PodManager, 'log') as mock_logger:
+        with mock.patch.object(
```

Comment:
notice a non-process safe counter makes this test flaky or impossible to do as the call of log.info happens in a process.